config/src/test/scala/com/typesafe/config/impl/ConfigTest.scala

```diff
@@ -1149,17 +1149,20 @@ class ConfigTest extends TestUtils {
     @Test
     def renderRoundTrip() {
         val allBooleans = true :: false :: Nil
+        val noMasking = Nil
```

Comment:
Supplied only Nil here for the roundtrip test. Unsure how to elegantly assert correctly when masking is enabled.